src/pydata_sphinx_theme/assets/styles/variables/_bootstrap.scss

```diff
@@ -15,3 +15,10 @@ $grid-breakpoints: (
   lg: 960px,
   xl: 1200px,
 );
+
+$dropdown-link-hover-bg: var(--pst-color-surface);
+// --pst-color-surface can also be assigned to the dark variant because it is
+// scoped to different values depending on light/dark theme
+$dropdown-dark-link-hover-bg: var(--pst-color-surface);
+$dropdown-link-active-bg: var(--pst-color-surface);
+$dropdown-dark-link-active-bg: var(--pst-color-surface);
```

Comment:
Not sure if this is the right move, but it seemed that if we want to override the default dropdown styles in this dropdown, the we'd probably want to override them in any other dropdown that we might add in the future?

Reply:
why do we need separate light/dark variables --- oh, is it because these are preexisting bootstrap variables that we're just overriding?

Reply:
correct